src/components/BookmarkCard/Details.tsx

```diff
@@ -53,7 +53,7 @@ export function Details({
                     href={href}
                     newTab={newTab}
                 >
-                    {stripTags(title)}
```

Comment:
Have you checked the commits history for when and why this was added?

Reply:
Yes, it was added in [this PR](https://github.com/prezly/content-renderer-react-js/pull/27).
The linked Linear ticket doesn't seem to have any discussion regarding stripping the tags 🤷🏻

Reply:
I believe it's this one, or similar:
https://linear.app/prezly/issue/DEV-6895/html-tags-in-story-bookmark-preview

Reply:
Another one:
https://linear.app/prezly/issue/CARE-97/campaign-emails-story-bookmark-embeds-display-html-code

Reply:
So before removing this, I'd check if the root cause is still there.

Additionally, I believe that even if the root cause might be fixed already, there might be a lot of existing faulty oEmbed records generated and persisted within existing Stories and Emaisl editor documents in the database. So it's worth checking too.

Reply:
I can see that both of these are not related to Content Renderer - one was solved on the backend, and another was an email renderer issue 🤔 

Also, they were both fixed after Dmitrii's PR, which makes me think that it was a workaround for the backend issue that Istvan fixed

Reply:
> [...] there might be a lot of existing faulty oEmbed records generated and persisted within existing Stories and Emails editor documents in the database. 

I'll make a migration to fix these.

Reply:
> I'll make a migration to fix these.

:heavy_check_mark: Done.

Also, the root cause seems to be fixed as well. 

So, I guess, it's safe to proceed now.